flask_rest_api/blueprint.py

```diff
@@ -136,12 +139,12 @@ def store_method_docs(method, function):
             for method in self.HTTP_METHODS:
                 if method in obj.methods:
                     func = getattr(obj, method.lower())
-                    store_method_docs(method, func)
+                    store_method_docs(method, func, sep=self.docstring_sep)
```

Comment:
Can't you access `self.docstring_sep` in `store_method_docs` itself?

Reply:
No, at least not with the base I'm working from. `store_method_docs` is defined within `Blueprint._store_endpoint_docs` and does not accept the instance as an arg.

My current base is a1c2dfc, v0.15.0.

Reply:
I'm not familiar with closures. I thought `self` would be accessible since it is defined in the scope of `Blueprint._store_endpoint_docs` where `store_method_docs` is defined.

Currently out of office, I'll try to check this and the rest when I come back.